packages/terra-disclosure-manager/src/terra-dev-site/doc/disclosure-manager/About.1.doc.mdx

```diff
@@ -0,0 +1,146 @@
+import { Badge } from 'terra-disclosure-manager/package.json?dev-site-package';
+import DisclosureManagerPropsTable from 'terra-disclosure-manager/src/DisclosureManager.jsx?dev-site-props-table';
+
+<Badge />
+
+# Terra Disclosure Manager
+
+The DisclosureManager is a stateful component used to manage disclosure presentation. It provides consumers with access to the `DisclosureManager` component, the `DisclosureManagerContext`, and the context wrapper `withDisclosureManager`. Additionally, it provides the `DisclosureManagerHeaderAdapter` to allow disclosed content to register header information with the disclosing container.
+
+For the purpose of clarity, the concepts explained can be broken down into:
+- Disclosure Manager
+- Disclosure Manager Children - content rendered as children of the DisclosureManager
+- Disclosed Content - content that will be disclosed by a DisclosureManager child
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-disclosure-manager`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+This package uses React's Context for inter-component communication. A single instance of this package must be installed to ensure that communication occurs between all Context Providers and Consumers. To help ensure a single instance is installed, this package should be defined as a peer dependency in packages that are reusable libraries or otherwise consumed by other packages.
+
+## Usage
+```js
+import DisclosureManager from 'terra-disclosure-manager';
+```
+
+## Component Features
+
+- [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
+- [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
```

Comment:
These links are broken, please update them to engineering site links.

Reply:
https://github.com/cerner/terra-framework/pull/1015/commits/5c7a10f353a6d5de8fd461983e6da383d1dbefa2